src/Chapter2.hs

```diff
@@ -740,7 +759,8 @@ the list with only those lists that contain a passed element.
 
 🕯 HINT: Use the 'elem' function to check whether an element belongs to a list
 -}
-contains = error "contains: Not implemented!"
+contains :: Int -> [[Int]] -> [[Int]]
+contains el l = filter (elem el) l
```

Comment:
Now, after you've mastered eta-reduction, you can apply such a technique to other functions as well  🙂 
But your implementation is already great 💯

Reply:
So easy to miss with my limited experience, thanks for pointing out.